apps/src/weblab/brambleHost.js

```diff
@@ -5,7 +5,8 @@
 (function () {
     "use strict";
 
-    require.config({
+    // for some reaosn, if we don't prepend with window here, webpack strips this
+    window.requirejs.config({
```

Comment:
This was the tricky bit. For some reason, webpack was stripping this block, but doesn't strip it when we use window.requirejs.